# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296241530 18000
#      Fri Jan 28 14:05:30 2011 -0500
# Branch release-3-4-x
# Node ID e4ed10d59f188196a6db991cbceab122f4b87d06
# Parent  35c55d1241c7acaa87cf62c2e0ed174b4c7ee3da
fix declaration of ETIME in arpack/util/second.f

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-28  John W. Eaton  <jwe@octave.org>
+
+	* arpack/util/second.f (ARSCND): Declare ETIME INTRINSIC, not EXTERNAL.
+
 2011-01-28  John W. Eaton  <jwe@octave.org>
 
 	* arpack: New directory.
 	* Makefile.am: Include arpack/module.mk.
 
 2011-01-26  John W. Eaton  <jwe@octave.org>
 
 	* mkf77def.in: Strip trailing whitespace.
diff --git a/libcruft/arpack/util/second.f b/libcruft/arpack/util/second.f
--- a/libcruft/arpack/util/second.f
+++ b/libcruft/arpack/util/second.f
@@ -16,17 +16,17 @@
 *     .. Local Scalars ..
       REAL               T1
 *     ..
 *     .. Local Arrays ..
       REAL               TARRAY( 2 )
 *     ..
 *     .. External Functions ..
       REAL               ETIME
-      EXTERNAL           ETIME
+      INTRINSIC          ETIME
 *     ..
 *     .. Executable Statements ..
 *
 
       T1 = ETIME( TARRAY )
       T  = TARRAY( 1 )
 
       RETURN
